src/acceptance-test/java/de/zalando/aruha/nakadi/repository/db/EventTypeDbRepositoryTest.java

```diff
@@ -108,7 +109,7 @@ public void whenUpdateExistingEventTypeItUpdates() throws NakadiException, Dupli
 
         repository.saveEventType(eventType);
 
-        eventType.setCategory("new-category");
+        eventType.setCategory(EventCategory.UNDEFINED);
```

Comment:
Why should this break the test? The original EventType had the category `UNDEFINED` and the updated one also.